101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json

```diff
@@ -0,0 +1,166 @@
+                                       },
+                                       "ResourceName":{
+                                          "type":"string"
+                                       }
+                                    },
+                                    "required":[
+                                       "Id",
+                                       "Name",
+                                       "ResourceId",
+                                       "ResourceName"
+                                    ],
+                                    "type":"object"
+                                 },
+                                 "type":"array"
+                              },
+                              "UserEmail":{
+                                 "type":"string"
+                              },
+                              "UserId":{
+                                 "type":"string"
+                              },
+                              "UserName":{
+                                 "type":"string"
+                              }
+                           },
+                           "type":"object"
+                        }
+                     },
+                     "operationOptions":"IncludeAuthorizationHeadersInOutputs"
+                  }
+               },
+               "actions":{
+                  "Condition":{
+                     "actions":{
+                        
+                     },
+                     "expression":{
+                        "and":[
+                           {
+                              "equals":[
+                                 "@triggerBody()?['Event']",
+                                 ""
+                              ]
+                           }
+                        ]
+                     },
+                     "runAfter":{
+                        
+                     },
+                     "type":"If"
+                  }
+               },
+               "outputs":{
+                  
+               }
+            },
+            "parameters":{
+               
+            },
+            "accessControl":{
+               "triggers":{
+                  "openAuthenticationPolicies":{
+                     "policies":{
+                        "ELMAuthPolicy":{
+                           "type":"AAD",
+                           "claims":[
+                              {
+                                 "name":"iss",
+                                 "value":"https://sts.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47/"
+                              },
+                              {
+                                 "name":"aud",
+                                 "value":"https://management.core.windows.net/"
+                              },
+                              {
+                                 "name":"appid",
+                                 "value":"810dcf14-1858-4bf2-8134-4c369fa3235b"
```

Comment:
what app is this?  (add details to the readme)

Reply:
This is azure ELM first party app Id will explain in Read me about this